CrashLib/CrashLib.xcodeproj/project.pbxproj

```diff
@@ -934,6 +934,11 @@
 		B21A71E41DD2A38C0044BB1C /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = (
```

Comment:
Can we add a new xcconfig for `CrashLib`?

Reply:
I'll create a separate bug for this.